model.js

```diff
@@ -48,3 +48,25 @@ exports.returnReviewComments = (reviewId) => {
     return Promise.reject({ status: 400, msg: err });
   }
 };
+
+exports.postNewComment = (params, reviewId) => {
+  if (typeof +reviewId === 'number') {
+    return db.query(`SELECT * FROM reviews;`).then((response) => {
```

Comment:
It may be beneficial to extract a function which checks if something exists (like you've checked if this review exists) into a separate utility function 🙂

Reply:
Great idea. There is actually already the function in models to return all reviews. I cannot figure out how to do it though as it seems to always be a returning promise. Would like a chat about this at your next convenience. No rush